Dockerfile

```diff
@@ -1,10 +1,7 @@
-FROM ubuntu:bionic
+FROM ubuntu:bionic AS builder
 
 # Install build dependencies
-RUN apt-get update
-RUN apt-get -y install software-properties-common
-RUN add-apt-repository -y ppa:beineri/opt-qt-5.15.2-bionic
-RUN apt-get -y install build-essential mesa-common-dev qt515base qt515serialport qt515svg cmake mercurial subversion git wget libfuse2
+RUN apt-get update && apt-get install -y software-properties-common && add-apt-repository -y ppa:beineri/opt-qt-5.15.2-bionic && apt-get install -y build-essential mesa-common-dev qt515base qt515serialport qt515svg cmake mercurial subversion git wget libfuse2 libgl1-mesa-dev
```

Comment:
Thanks for explaining the purpose of this change. Using multiple lines with `\` would be nice from readability perspective.